### ISSUE TITLE
Disable/hide close icon `x` in titles for tiled windows

### ISSUE THREAD

**Issue by joelostblom** [pop-os/shell#1518]
**(1) Issue/Bug Description:**

It seems like there was a recent update to the appearance of the title bar for tiling windows, which introduce a close icon `x`. This icon is rather big and not very aesthetically pleasing, and since it is right aligned it looks like it belongs to the neighboring window which is a bit confusing as can be seen below:

![image](https://user-images.githubusercontent.com/4560057/189213284-0d170945-3204-4e5c-8798-ddcf431c7651.png)

**(2) Steps to reproduce (if you know):**

Enable tiled windows and open more than one

**(3) Expected behavior:**

I would expect there to be an option/config file to disable this close icon (like there is in i3).

**(4) Distribution (run `cat /etc/os-release`):**
 
```
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

```

**(5) Gnome Shell version:**

42.1

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```1.1.0```

**(7) Where was Pop Shell installed from:**

Came with system76 computer

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**

Single 1080p laptop

**(9) Other Installed/Enabled Extensions:**

**(10) Other Notes:**

**Comment by Avijneyam** [comment]
Yes i agree with you close icon should be only in active one not all tilted windows

**Comment by mmstick** [comment]
That wouldn't make sense if you wanted to close a non-active window, as is usually the case. The web browser you're using has a close icon on each of its non-active tabs, too.

**Comment by joelostblom** [comment]
Yes it probably makes sense to keep them there as a default but making them more muted. In we browsers they often blight up on hover but otherwise don't stand out that much.

Is there a config file to edit to disable these currently?

**Comment by mmstick** [comment]
There is not
